packages/react/src/runtime/ax.ts

```diff
@@ -41,7 +48,16 @@ export default function ax(classNames: (string | undefined | null | false)[]): s
       continue;
     }
 
-    const groups = cls.split(' ');
```

Comment:
What if:

```
const groups = (cls + '').split(' ');
```

Reply:
Yeah actually, doing `const groups = String(cls).split(' ');` does work actually, just feels like it doesn't tell you what it's doing is all.

Reply:
I'd do whatever results in better performance.

Reply:
Well, not sure about the array syntax `<Component xcss={[style, style]} />`, but whatever, that fails types anyways.

Reply:
Doing arbitrary online benchmarks which mean little, protecting via a typeof like:
```tsx
if (typeof c1 === 'string') {
	c1.split(' ')
} else {
	String(c1).split(' ')
}
```
is faster by ~4% than just blindly calling `String(c1)` first.  I gather `String()` doesn't handle this (but could/should).

Worst-case, I'd expect it to be 1:1, I don't see how v8 could do anything super special here—basically the negative impact is just from doing `typeof` checks, which are pretty minimal.

Reply:
Sweet ok. I think we can lean on type assumptions and safely use the typeof check tbh.